react-native-libraries.json

```
@@ -14395,5 +14395,13 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/your-username/react-native-country-flag",
```

Hey @firashelou, thanks for adding new entry to the directory! 👍 

Looks like the set package GitHub repository is incorrect, which is the reason for CI failure. According to npm registry entry, this is the correct repository:

```suggestion
    "githubUrl": "https://github.com/YannisHofmann/react-native-country-flag",
```